examples/user_server_test.rs

```diff
@@ -124,6 +124,92 @@ pub async fn get_user() {
     };
 }
 
+/// Test for the DELETE route.
+///
+/// We add a new user to the database, then delete it and ensure that the
+/// server returns a 200 status code.
+///
+/// We then try to delete the same user again and ensure that the server
+/// returns a 404 status code.
+#[tokio::test]
+pub async fn delete_user() {
+    // Create a new Request object, just as we did for the put_user test.
+    let request = Request::post("users").with_body(UserInput {
+        year_of_birth: 2000,
+    });
+
+    // Similarly, execute the said request and get the output.
+    let user = CONTEXT
+        .run(request)
+        .await
+        .expect_status(StatusCode::CREATED)
+        .await;
+
+    assert_body_matches! {
+        user,
+        User { id, year_of_birth: 2000 },
+    };
+
+    let request = Request::delete(path!["users", id]).with_body(());
+
+    CONTEXT
+        .run(request)
+        .await
+        .expect_status::<String>(StatusCode::OK)
+        .await;
+
+    // We try to delete the same user again and ensure that the server
+    // returns a 404 status code.
+    let request = Request::delete(path!["users", id]).with_body(());
+
+    CONTEXT
+        .run(request)
+        .await
+        .expect_status::<String>(StatusCode::NOT_FOUND)
+        .await;
+}
+
+/// Test for the PUT route.
+///
+/// We add a new user to the database, then update its profile and ensure that
+/// the server returns a 200 status code.
+#[tokio::test]
+pub async fn put_user() {
+    // Create a new Request object, just as we did for the put_user test.
```

Comment:
seems that you reference the put_user test, as if it was the first, but the first is the post_user